bundle/src/projects/common/modules/experiments/tests/eager-prebid.ts

```diff
@@ -0,0 +1,24 @@
+import type { ABTest } from '@guardian/ab-core';
+import { bypassMetricsSampling } from '../utils';
+
+export const eagerPrebid: ABTest = {
+	id: 'EagerPrebid',
+	author: '@commercial-dev',
+	start: '2023-03-13',
+	expiry: '2023-03-30',
+	audience: 5 / 100,
+	audienceOffset: 35 / 100,
+	audienceCriteria: 'All pageviews',
+	successMeasure:
+		'Ads lazy load faster, without affecting the page load time',
+	description: 'Test the impact of running prebid on page load',
+	variants: [
+		{ id: 'control', test: bypassMetricsSampling },
+		{ id: 'variant-20', test: bypassMetricsSampling },
```

Comment:
This one's just floating an idea and completely up to you, but I'd probably name these: `variant-200`, `variant-175`, `variant-150`, `variant-125`, `variant-100`

Reply:
I was considoring that, but I've already given them to Zoyah now🙈